apps/music/views/home/view.js

```diff
@@ -113,7 +115,8 @@ HomeView.prototype.search = function(query) {
   return document.l10n.formatValues(
     'unknownTitle', 'unknownArtist', 'unknownAlbum'
   ).then(([unknownTitle, unknownArtist, unknownAlbum]) => {
-    var albumResults = this.fetch('/api/search/album/' + query)
+    var albumResults = this.fetch('/api/search/album/' +
+                                  encodeURIComponent(query))
```

Comment:
Let's just do `query = encodeURIComponent(query);` at the top of this function so we're not calling it three separate times.